docs/dev/stream/state/broadcast_state.md

```diff
@@ -171,9 +171,8 @@ exposes some functionality which is not available to the `BroadcastProcessFuncti
   to register event and/or processing time timers. When a timer fires, the `onTimer()` (shown above) is invoked with an 
   `OnTimerContext` which exposes the same functionality as the `ReadOnlyContext` plus 
    - the ability to ask if the timer that fired was an event or processing time one and 
-   - to query the key associated with the timer.
-
-  This is aligned with the `onTimer()` method of the `KeyedProcessFunction`. 
+   - to query the key associated with the timer.<br/>
```

Comment:
That is not completely right. In the original code, the line break was forced by the empty white line after (nested) list. However, that causes the (outer) list enumeration to be broken; the items "_the ReadOnlyContext in the processElement()_" and "_the Context in the processBroadcastElement()_" both have a numeration of "_1_" in the original code.

So to solve the enumeration problem, we have to take out the empty white line. That however implies that we have to include an explicit line break.

Reply:
You are right! I only paid attention to the line break.

Reply:
This part still doesn't look correct.
![untitled](https://user-images.githubusercontent.com/5725237/44080974-dce6f1e0-9fad-11e8-8ea1-ae98c5254b19.png)

It's now hard to tell whether `This is aligned with the onTimer() method of the KeyedProcessFunction.` applies to `the ReadOnlyContext in the processElement() method ...` or either/both of the sub-items.

We should just re-phrase this part. Actually I would just remove it; as someone not deeply familiar with process functions I simply assumed that `KeyedBroadcastProcessFunction#onTimer` works exactly like `KeyedProcessFunction#onTimer`.

Reply:
I totally agree, @zentol. To avoid cluttered commits in which we (a) fix something and (b) improve something, I kept this commit as is (i.e., fixing the list/rendering issue), and added a second commit in which I remove the ambiguous line from the text as you proposed. The new commit is 75d815bb4240137016daf19eeb3922aa31da53a1.